Update read me for the custom translation section. (#53)
onfido/react-native-sdk: README.md

### 5. Localization

The SDK supports and maintains the following 6 languages:

 - English (en) 🇬🇧
 - Spanish (es) 🇪🇸
 - French (fr) 🇫🇷
 - German (de) 🇩🇪
 - Italian (it) 🇮🇹
 - Portuguese (pt) 🇵🇹
 
However, you can add your own translations.

#### Android

By default, custom localisation is enabled on Android. There is no configuration needed on React Native SDK to enable it.
You could also provide custom translation for a locale that we don’t currently support, by having an additional XML strings file inside your resources folder for the desired locale. See [Localisation section of Android SDK repo](https://github.com/onfido/onfido-android-sdk#4-localisation) for the details.

#### iOS

> There is simple configuration needed on React Native SDK to enable custom localisation.

Reviewer: ```suggestion
There is a simple configuration needed on the React Native SDK to enable custom localisation.
```

> You could also provide custom translation for a locale that we don’t currently support

Reviewer: ```suggestion
You can also provide a custom translation for a locale that Onfido doesn't currently support.
```

Reviewer: This should be the first sentence under #### iOS

Reviewer: Also need to swap these 2 sentences around, otherwise doesn't really make sense

Author: fixed.

1. Add this statement to your configuration object.
```
localisation: {
  ios_strings_file_name: '<Your .strings file name in iOS app bundle>',
},
```

> 2. ```cd ios``` folder, and open your XCode workspace.

Reviewer: Not sure it really makes sense to have ```cd ios``` as the instruction here. It would be better to write it in english and then have the code as an example. So for example "Navigate to the iOS folder" or similar

Author: done.

> 3. Follow the instruction here to add a new custom language or override existing translations [Localisation section of iOS SDK repo](https://medium.com/lean-localization/ios-localization-tutorial-938231f9f881).

Reviewer: ```suggestion
3. Follow the instructions in the [Localisation section of iOS SDK repo](https://medium.com/lean-localization/ios-localization-tutorial-938231f9f881) to add a new custom language or override existing translations.
```

> 3. Follow the instructions in the [iOS Localisation](https://medium.com/lean-localization/ios-localization-tutorial-938231f9f881) to add a new custom language or override existing translations.

Reviewer: ```suggestion
3. Follow the instructions for [iOS Localisation](https://medium.com/lean-localization/ios-localization-tutorial-938231f9f881) to add a new custom language or override existing translations.
```

> 4. [Here is the keys need to be translated](https://github.com/onfido/onfido-ios-sdk/blob/master/localization/Localizable_EN.strings).

Reviewer: ```suggestion
4. [Here are the keys that need to be translated](https://github.com/onfido/onfido-ios-sdk/blob/master/localization/Localizable_EN.strings).
```

> 4. [Here are the keys that need to be translated in iOS SDK repo](https://github.com/onfido/onfido-ios-sdk/blob/master/localization/Localizable_EN.strings).

Reviewer: ```suggestion
4. You can find the keys that need to be translated in the [iOS SDK repo](https://github.com/onfido/onfido-ios-sdk/blob/master/localization/Localizable_EN.strings).
```
